api/cloudformation.yml

```diff
@@ -0,0 +1,144 @@
+Parameters:
+
+  JobQueueArn:
+    Type: String
+
+  RtcGammaJobDefinition:
+    Type: String
+
+  AuthPublicKey:
+    Type: String
+
+  AuthAlgorithm:
+    Type: String
+    Default: RS256
+
+Outputs:
+
+  Url:
+    Value: !Sub "https://${RestApi}.execute-api.${AWS::Region}.amazonaws.com/${Stage}/"
+
+Resources:
+
+  RestApi:
+    Type: AWS::ApiGateway::RestApi
+    Properties:
+      Name: !Ref AWS::StackName
```

Comment:
we could describe the entire api using the `body` attribute and an openapi spec instead of the separate `Resource` and `Method` items, not sure which I like